utils/scripts/install-el8.sh

```diff
@@ -55,6 +56,7 @@ dnf --nodocs install \
     python3-devel \
     python3-pip \
     sg3_utils \
+    sudo \
```

Comment:
I should probably have done a review rather than comments at this point but this will be the last one.

We've been trying to remove sudo from the dockerfiles yet this PR adds it universally and I can't obviously see where it's used?

Reply:
A big part of the Dockerfile (and helper scripts) changes were to bring them into sync with each other so that if you open them all, for example with `vim -d` the only differences between them are the things that truly need to be different due to O/S differences, etc.

`sudo` was added here because it was present in another `install-*.sh`.  IIRC, it was in the `el7` and given that all of our Unit Testing is on EL7 it was presumed to be canonical and that EL8 would need the same packages installed if/when UT is moved there, and `sudo` would be one of the packages that needed adding.

Reply:
True, and sudo was only present on el7 because it runs the fault injection tests in docker.  Given we want to move that to el8 and remove el7 entirely then having it universal isn't that big of an issue.

Reply:
Or just given that we should be able to run FI in docker on any platform, they all need to have `sudo` available.

Reply:
none would be preferred but I was unable to achieve that without hitting the "files owned by root" issue with the Jenkins workspace.